src/library_html5.js

```diff
@@ -1557,7 +1557,7 @@ var LibraryJSEvents = {
   },
 
   emscripten_set_pointerlockchange_callback: function(target, userData, useCapture, callbackfunc) {
-    if (!document.body.requestPointerLock && !document.body.mozRequestPointerLock && !document.body.webkitRequestPointerLock && !document.body.msRequestPointerLock) {
+    if (!document.body || (!document.body.requestPointerLock && !document.body.mozRequestPointerLock && !document.body.webkitRequestPointerLock && !document.body.msRequestPointerLock)) {
```

Comment:
That's a good point. Added a check for the `document` object as well. I'll revisit this topic with the new multithreading aware js library proxying model once I have time to take that further.